mPowerSDK/TasksAndSteps/APHActivityManager.m

```diff
@@ -136,7 +136,12 @@ + (instancetype)defaultManager
     }
     
     // Replace the language in the last step
-    [task.steps.lastObject setTitle:NSLocalizedStringWithDefaultValue(@"APH_ACTIVITY_CONCLUSION_TEXT", nil, APHLocaleBundle(), @"Thank You!", @"Main text shown to participant upon completion of an activity.")];
+    if ([surveyId isEqualToString:APHMemoryActivitySurveyIdentifier]) {
```

Comment:
No, the last step is always the conclusion step, it just has different language. "Good Job!" instead of "Thank You!"